install.ps1

```diff
@@ -78,6 +78,17 @@ $desktopPath = [Environment]::GetFolderPath("Desktop")
 Set-Location -Path $desktopPath -PassThru | Out-Null
 
 if (-not $noChecks.IsPresent) {
```

Comment:
Will it be better if the PowerShell version check is outside the skip parameter so it can't be skipped? I think that would at the very least lead to less chaos at later stages.